### PR TITLE
add and update assets

### PR DESCRIPTION
**Pull request by ronitmndl**
1. Fix formatting of cerebras-GPT size explanation
2. Some standardisation edits

Add previously undocumented dependencies:

1. Create kakaobrain.yaml; Add `COYO-700M` dataset
2. Create oxford.yaml; Add `WebVid-10M` and `WebVid-2M `datasets
3. Create stonybrook.yaml; Add `SBU Captions` dataset
4. Add `LAION-400M` and `LAION-2B-en` datasets to laion.yaml
5. Add `LAION-1B` dataset to alibaba.yaml
6. Create ens.yaml; Add `HowTo100M` dataset
7. Add `Conceptual Captions` and `Conceptual 12M` datasets to google.yaml

Add new assets:
1. Create bloomberg.yaml; Add `BloombergGPT` model and `FinPile` dataset
2. Add `Microsoft Security Copilot`
3. Add AssemblyAI's `Conformer-1` model
4. Create adept.yaml; Add `ACT-1` model
5. Add Meta's `SAM` model and `SA-1B` dataset ✍️

### CODE REVIEW

File: assets/bloomberg.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      They compare BloombergGPT to the three closest models: GPT-NeoX, OPT-66B and
+      BLOOM-176B. They also report results from the original GPT-3 whenever externally
+      available. They conclude "We achieve strong results on general LLM benchmarks
+      and outperform comparable models on financial tasks. We attribute this, in
+      decreasing order of impact, to 1. a well-curated internal dataset, 2. our
+      unique choice in tokenizer, and 3. an up-to-date architecture."
+    explanation: See [[Evaluation]](https://arxiv.org/pdf/2303.17564.pdf#section.5)
+  size: 50B parameters (dense model)
+  # Construction
+  dependencies: [FinPile, The Pile, C4, Wikipedia]
```
Comment (ronitmndl) [inline assets/bloomberg.yaml:87]: They mention this in the paper: "Our model is a decoder-only causal language model based on BLOOM (Scao et al., 2022)." and "In developing BloombergGPT, we benefited from existing code developed as part of the BLOOM effort Scao et al. (2022), showing that a moderately sized team can produce a competitive model on domain-specific data.". 

Should BLOOM be added as a dependency?

File: assets/perplexity.yaml [hunk 2]
```diff
@@ -49,7 +49,10 @@
   organization: Perplexity
   description: Perplexity Ask is a new search interface that uses advanced artificial
     intelligence technologies
-  created_date: unknown
+  created_date:
+    value: 2022-12-07
+    explanation: Perplexity Ask was announced in a [[Twitter post]](https://twitter.com/perplexity_ai/status/1600551871554338816?s=20)
+      by the company.
   url: https://www.perplexity.ai/
   dependencies: [Perplexity model]
```
Comment (ronitmndl) [inline assets/perplexity.yaml:57]: The [announcement post](https://twitter.com/perplexity_ai/status/1600551871554338816) said "Announcing Perplexity Ask, a new search interface that uses OpenAI GPT 3.5 and Microsoft Bing to directly answer any question you ask."

Should dependencies be changed to [GPT 3.5, Bing Search] ?

**Comment by ronitmndl** [comment]
Requesting review @rishibommasani

File: assets/meta.yaml [hunk 3]
```diff
@@ -410,3 +410,190 @@
...
+  description: A foundation model for image segmentation.
+  created_date:
+    value: 2023-04-05
+    explanation: The date the [[Meta blog post]](https://ai.facebook.com/blog/segment-anything-foundation-model-image-segmentation/)
+      was released.
+  url: https://arxiv.org/pdf/2304.02643.pdf
+  model_card:
+    value: https://arxiv.org/pdf/2304.02643.pdf#page=28
+    explanation: Model card can be found in the Appendix section of the paper.
+  modality: image, segmentation prompt (point, box, mask and text)
```
Comment (ronitmndl) [inline assets/meta.yaml:528]: Unsure about this, should it just be image, text?

File: assets/meta.yaml [hunk 4]
```diff
@@ -410,3 +410,190 @@
...
+      slightly below that of the manually annotated ground truth.
+      Second, we find consistently strong quantitative and qualitative results on
+      a variety of downstream tasks under a zero-shot transfer protocol using prompt
+      engineering, including edge detection, object proposal generation, instance
+      segmentation, and a preliminary exploration of text-to-mask prediction."
+    explanation: See [[Zero-Shot Transfer Experiments]](https://arxiv.org/pdf/2304.02643.pdf#section.7)
+      for more details.
+  size: unknown
+  # Construction
+  dependencies: [SA-1B]
```
Comment (ronitmndl) [inline assets/meta.yaml:543]: In the paper they mention "Motivated by scalability and access to strong pre-training, we use an MAE [47] pre-trained Vision Transformer (ViT) [33] with minimal adaptations to process high resolution inputs, specifically a ViT-H/16 with 14×14 windowed attention and four equally-spaced global attention blocks, following [62]". They also use ViT-B/L, but "Unless otherwise specified: (1) SAM uses an MAE [47] pre-trained ViT-H [33] image encoder"

should ViT-H be added as a dependency?

File: assets/aleph_alpha.yaml [hunk 5]
```diff
@@ -41,7 +41,9 @@
   modality: text (English, French | français, German | Deutsch, Italian | italiano,
     Spanish | Español)
   analysis: ''
-  size: 200B parameters (dense model) according to [[this news article]](https://www-heise-de.translate.goog/news/Machine-Learning-Aleph-Alpha-feilt-mit-Oracle-und-Nvidia-an-transformativer-KI-6269269.html?_x_tr_sl=de&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=sc)
+  size:
+    value: 200B parameters (dense model)
```
Comment (rishibommasani) [inline assets/aleph_alpha.yaml:45]: Nice catch, thanks!

File: assets/ai21.yaml [hunk 6]
```diff
@@ -79,7 +79,7 @@
   model_card: none
   modality: text (English)
   size:
-    value: 178B parameters (dense)
+    value: 178B parameters (dense model)
```
Comment (rishibommasani) [inline assets/ai21.yaml:82]: Good catch, thanks!

File: assets/bloomberg.yaml [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+  size:
+    value: 363B tokens
+    explanation: See [[Financial Datasets]](https://arxiv.org/pdf/2303.17564.pdf#subsection.2.1)
+  sample: []
+  analysis: ''
+  # Construction
+  dependencies: []
+  license: unknown
+  included:
+    value: |
```
Comment (rishibommasani) [inline assets/bloomberg.yaml:28]: Just curious: what does the `|` syntax do compared to `>`?

File: assets/bloomberg.yaml (hunk 1 above)
Comment (rishibommasani) [reply assets/bloomberg.yaml:87]: As I understand, they are using the BLOOM codebase as opposed to the BLOOM model so no, I think what you have is correct.

File: assets/laion.yaml [hunk 8]
```diff
@@ -1,5 +1,59 @@
...
+    explanation: The date the [[blog post]](https://laion.ai/blog/laion-400-open-dataset/)
+      was released.
+  url: https://laion.ai/blog/laion-400-open-dataset/
+  datasheet: https://laion.ai/blog/laion-400-open-dataset/
+  modality: text (English) and image
+  size: 400M image-text pairs
+  sample: []
+  analysis: ''
+  # Construction
+  dependencies: [CommonCrawl]
```
Comment (rishibommasani) [inline assets/laion.yaml:21]: should there be a dependency on CLIP due to the filtering?

File: assets/laion.yaml [hunk 9]
```diff
@@ -49,3 +103,54 @@
...
+      released. Note that the dataset was already compiled earlier, e.g. the Stable
+      Diffusion model released earlier was trained on a subset of LAION-5B
+  url: https://arxiv.org/pdf/2210.08402.pdf
+  datasheet: https://laion.ai/blog/laion-5b/
+  modality: text (English) and image
+  size: 2.32B image-text pairs
+  sample: []
+  analysis: ''
+  # Construction
+  dependencies: [LAION-5B]
```
Comment (rishibommasani) [inline assets/laion.yaml:125]: should there be dependencies on clip and mclip due to filtering?

File: assets/meta.yaml (hunk 3 above)
Comment (rishibommasani) [reply assets/meta.yaml:528]: i think image, text is fine. And the segmentation info could go into the description.

File: assets/meta.yaml (hunk 4 above)
Comment (rishibommasani) [reply assets/meta.yaml:543]: Good question, I think we should say ViT-H if it is a standard version of ViT-H, but if the pretrained it themselves with their own data, then no. Will look at the paper again

File: assets/microsoft.yaml [hunk 10]
```diff
@@ -296,8 +296,8 @@
   size: 1.6B parameters (dense model)
   dependencies:
     - The Pile
-    - Common Crawl
-    - LAION-2B
+    - CommonCrawl
```
Comment (rishibommasani) [inline assets/microsoft.yaml:299]: good catch!

File: assets/openai.yaml [hunk 11]
```diff
@@ -418,11 +418,11 @@
   # General
   organization: OpenAI
   description: ''
-  created_date: 2019-11
+  created_date: Nov 2019
```
Comment (rishibommasani) [inline assets/openai.yaml:421]: I forget, but do partial dates break the sorting. If we don't know the day of month, I think it is fine to say Nov 1, 2019 and include in descritpion we assumed to be Nov 1

File: assets/perplexity.yaml (hunk 2 above)
Comment (rishibommasani) [reply assets/perplexity.yaml:57]: yes i think so

Comment (rishibommasani) [reply assets/perplexity.yaml:57]: i.e. i think you can remove the dataset + model nodes for perplexity and just have application with these dependencies.

**Review by rishibommasani — CHANGES_REQUESTED** [review]
Amazing work!!!! Left a few comments, but looks fantastic!

File: assets/bloomberg.yaml (hunk 7 above)
Comment (ronitmndl) [reply assets/bloomberg.yaml:28]: I used it to create bulleted/numbered lists like this:
image

`|` is supposed to preserve all newline characters, unlike `>` which converts newlines to spaces, but it only seemed to work for lists here unsure why.

Comment (rishibommasani) [reply assets/bloomberg.yaml:28]: Ah got it, good to know!

File: assets/laion.yaml (hunk 8 above)
Comment (ronitmndl) [reply assets/laion.yaml:21]: You're right, will add CLIP as a dependency

File: assets/laion.yaml (hunk 9 above)
Comment (ronitmndl) [reply assets/laion.yaml:125]: You're right, but I think only CLIP for LAION-2B-en as it's english-only dataset, "cosine between text and image ViT-B/32 embeddings, clip for en, mclip for multi and nolang". However both CLIP and mCLIP should be added as dependencies of LAION-5B.

File: assets/openai.yaml (hunk 11 above)
Comment (ronitmndl) [reply assets/openai.yaml:421]: I don't think they break sorting, they are automatically placed at the end of that month (for MM-YY) or year for (YY).

Yes that would work, will update all the partial dates accordingly

File: assets/meta.yaml (hunk 4 above)
Comment (ronitmndl) [reply assets/meta.yaml:546]: From what I understand, standard version of ViT-H would be the one presented in the original [ViT paper](https://paperswithcode.com/paper/an-image-is-worth-16x16-words-transformers-1), but here they have used a ViT-H model that's pre-trained using the Masked Autoencoder approach from [this paper](https://arxiv.org/abs/2111.06377)

**Review by rishibommasani — APPROVED** [review]
Fantastic, thanks!
